src/esn.calendar.libs/app/components/partstat-buttons/partstat-buttons.controller.js

```diff
@@ -38,9 +38,9 @@ function CalPartstatButtonsController($attrs, session, calEventService, calEvent
 
     if ($attrs.changePartstat) {
       self.changePartstat({ partstat: partstat });
-    } else {
-      _changeParticipation(partstat);
     }
+
+    _changeParticipation(partstat);
```

Comment:
If `else` block is removed, so it means the code block always run this method, is this expected?